include/deal.II/lac/precondition.h

```diff
@@ -1966,16 +1966,34 @@ class PreconditionChebyshev : public Subscriptor
       power_iteration
     };
 
+    /**
+     * An enum to define the available types of polynomial types.
+     */
+    enum PolynomialType
```

Comment:
if EigenvalueAlgorithm is an `enum class` then could we also make this an `enum class`? I'd prefer to keep the enum types consistent.

Reply:
@drwells Done!